src/HappyCode.NetCoreBoilerplate.Api/appsettings.json

```diff
@@ -1,16 +1,33 @@
 {
-  "Logging": {
-    "LogLevel": {
-      "Default": "Warning"
-    }
-  },
-
   "ConnectionStrings": {
     "SimpleMySqlDb": "server=mysql;Database=employees;Uid=user;Pwd=simplepwd;"
   },
 
   "PingWebsite": {
     "Url": "https://kurzyniec.pl",
     "TimeIntervalInMinutes": 60
+  },
+  "Serilog": {
+    "MinimumLevel": "Information",
+    "WriteTo": [
+      {
+        "Name": "Async",
+        "Args": {
+          "configure": [
+            {
+               "Name": "Console"
+            },
+            {
+              "Name": "File",
```

Comment:
You would be surprised how common is the scenario when the apps are logging to file sinks and the third party components (like Filebeat) are harvesting and forwarding data to a centralized log system. This pattern helps to avoid log data loss in the event of a broken network connection as well as add support for a centralized log system without introducing changes into existing services.

Reply:
Now I got it. I'm used to use ELK stack, but the logs are not so crucial.

Reply:
Logs are never crucial until you have an issue on the production ;)

Reply:
You are right! We use AWS, so we rely on them ;)